eventdata/parameter_sources/elasticlogs_kibana_source.py

```diff
@@ -75,10 +75,12 @@ class ElasticlogsKibanaSource:
                                     or relative. Defaults to '1d'.
                                         '4d' - Consists of a number and either m (minutes), h (hours) or d (days). Can not be lower than 1 minute.
                                         '10%' - Length given as percentage of window size. Only available when fieldstats_id have been specified.
-        "discover_size"         -   Nunmber of documents to return in Discover. Defaults to 500.
+                                        'random' - Length is randomized between START and END interval. Only available when fieldstats_id have been specified.
```

Comment:
Should we clarify the data type? AFAICS the default is not 500 but `1d` so this should be a string anyway right?

Reply:
I think the diff displayed the change a bit weirdly, `default 500` is for `discover_size` which I haven't changed at all and it's the line bellow. My change is adding `random` as a possible value for window_length, which is slightly above but not included in the preview in the diff.

Reply:
Ah right I see now sorry!